Web2 vs Web3 Docs Readability Improvements [Fixes #8991] (#8992)
ethereum/ethereum-org-website: src/content/developers/docs/web2-vs-web3/index.md

lang: en
---

> Web2 refers to the version of the internet that most of us are familiar with today. An internet where access to services requires your personal data to be custodied by companies whose profit motives strongly incentivises data monetisation. Web3, in the context of Ethereum, refers to decentralized apps that utilises the self-custodial guarantees made possible through the blockchain. These are apps that provide similar, or even brand new, services without requiring the user to ever give up ownership of their own data.

Reviewer: ```suggestion
Web2 refers to the version of the internet most of us know today. An internet where access to services requires companies to take custody of your personal data, leading to centralization. Web3, in the context of Ethereum, refers to decentralized apps that utilize the self-custodial guarantees made possible through the blockchain. These are apps that provide similar, or even brand new, services without requiring the user to ever give up ownership of their own data.
```

Cleaning this up to center around data custody more

|       Servers for gig-economy apps could go down and affect worker income       | Web3 servers can't go down – they use Ethereum, a decentralized network of 1000s of computers as their backend |

This doesn't mean that all services need to be turned into a dapp. These examples are illustrative of the main differences between web2 and web3 services.

## Web3 limitations {#web3-limitations}

Web3 has some limitations right now:

- Scalability – Transactions are slower on Web3 because decentralisation necessitates a consensus mechanism. Changes to state, like a payment, need to be processed by a node and propagated throughout the network.
- UX – Interacting with Web3 applications might require extra steps, software, and education. While the experience gap between Web2 and Web3 is rapidly shrinking, this can be a hurdle to adoption as there are some fundamental differences which requires getting used to.
- Accessibility – The lack of integration in modern web browsers makes Web3 less accessible to most users.

> - Cost – Blockchain memory space as well as computing resources are relatively more expensive resulting in only critical components of a dapp being hosted on the chain.

Reviewer: ```suggestion
- Cost – gas fees can make transactions and deploying smart contracts to mainnet expensive at times.
```

This confused me, but I think this improves this point

## Centralization vs decentralization {#centralization-vs-decentralization}

The table below summarizes, in broad-strokes, the advantages and disadvantages of centralized and decentralized digital networks.

| Centralized Systems                                                                                                                                                                                                    | Decentralized Systems                                                                                                                                                                                                            |
| ---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | -------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |

> | Low network diameter (all participants are connected to a central authority); information propagates quickly, as propagation is handled by a central authority with lots of computational resources.                   | The furthest participants on the network may potentially be many edges away from each other. Information broadcasted from one side of the network may take a long time to reach the furthest edges.                              |

Reviewer: ```suggestion
| Low network diameter (all participants are connected to a central authority); information propagates quickly, as propagation is handled by a central authority with lots of computational resources.                   | The furthest participants on the network may potentially be many edges away from each other. Information broadcast from one side of the network may take a long time to reach the furthest edges.                              |
```

> | Usually higher performance (higher throughput, fewer total computational resources expended) and easier to implement as no consensus is required.                                                                      | Usually lower performance (lower throughput, more total computational resources expended) and more complex to implement due to complexities in achieving consensus.                                                              |

Reviewer: ```suggestion
| Usually higher performance (higher throughput, fewer total computational resources expended) and easier to implement.                                                                      | Usually lower performance (lower throughput, more total computational resources expended) and more complex to implement.                                                              |
```

Consensus isn't really the only thing here so this seems unnecessary

> | In the event of conflicting data, resolution is straightforward: the ultimate source of truth is the central authority.                                                                                                | A protocol (often with its own consensus mechanism and rules) is needed for dispute resolution in the event of conflicting claims about the state of data which the network should be synchronized on.                           |

Reviewer: ```suggestion
| In the event of conflicting data, resolution is clear and easy: the ultimate source of truth is the central authority.                                                                                                | A protocol (often with its own consensus mechanism and rules) is needed for dispute resolution, if peers make conflicting claims about the state of data which participants are meant to be synchronized on.                        |
```

This doesn't really change what was here already and would knock back translations.

> | Single point of failure: malicious actors may be able to take down the network by targeting the central authority.                                                                                                     | No single point of failure: network can still function as long as honest participants continue to form the majority of the network.                                                                                              |

Reviewer: ```suggestion
| Single point of failure: malicious actors may be able to take down the network by targeting the central authority.                                                                                                     | No single point of failure: network can still function even if a large proportion of participants are attacked/taken out.                                                                                            |
```

Theres attack vectors that occur at <50% https://ethereum.org/en/developers/docs/consensus-mechanisms/pos/attack-and-defense/#attackers-with-33-stake so I think we should keep whats there before

| Central authority can censor data, potentially cutting off parts of the network from interacting with the rest of the network.                                                                                         | Censorship is much harder, as information has many ways to propagate across the network.                                                                                                                                         |
| Participation in the network is controlled by the central authority.                                                                                                                                                   | Anyone can participate in the network; there are no “gatekeepers.” Ideally, the cost of participation is very low.                                                                                                               |

> Note that these are general patterns that may not hold true in every network. Furthermore, in reality the degree to which a network is centralized/decentralized lies on a spectrum; no network is entirely centralized or entirely decentralized. Critically, the extent of decentralisation should instead be informed by the aggregate of individual choices, each determining what suits them best without the coercive forces of scale.

Reviewer: ```suggestion
Note that these are general patterns that may not hold true in every network. Furthermore, in reality the degree to which a network is centralized/decentralized lies on a spectrum; no network is entirely centralized or entirely decentralized.
```

This addition seems unnecessary and adds some confusion for me
